### PR TITLE
Add messaging to messaging module

### PR DESCRIPTION
**Pull request by KPostOffice**
See package update and advise reporter.  We are moving the `messaging` module that is repeated there to the central `thoth-messaging`, here.

This will likely introduce breaking changes to anyone repositories using the `thoth-messaging` repo currently

### CODE REVIEW

**Comment by pacospace** [comment]
> See package update and advise reporter. We are moving the `messaging` module that is repeated there to the central `thoth-messaging`, here.
> 
> This will likely introduce breaking changes to anyone repositories using the `thoth-messaging` repo currently

Why don't we keep the basic message here only and the other messages specific to the components  stay in the specific component, what we would import is `thoth.messaging.message_base` so we don't need to change thoth-messaging every time we modify only a specific component. We also reduce the number of releases which might be required for one single component. WDYT? @KPostOffice @fridex @goern

**Comment by KPostOffice** [comment]
> > See package update and advise reporter. We are moving the `messaging` module that is repeated there to the central `thoth-messaging`, here.
> > This will likely introduce breaking changes to anyone repositories using the `thoth-messaging` repo currently
> 
> Why don't we keep the basic message only here and the other messages specific to the components stay in the specific component, what we would import is `thoth.messaging.message_base` so we don't need to change thoth-messaging every time we modify only a specific component. WDYT? Or here we want to have one source for all messages? @KPostOffice @fridex @goern

I was wondering the same thing.  Right now I have a naming convention for the messages so that it is obvious which package that the messages belong to.  My only thought is if we end up needing a message in multiple repos then we'd have to reuse code between them.

**Comment by pacospace** [comment]
> > > See package update and advise reporter. We are moving the `messaging` module that is repeated there to the central `thoth-messaging`, here.
> > > This will likely introduce breaking changes to anyone repositories using the `thoth-messaging` repo currently
> > 
> > 
> > Why don't we keep the basic message only here and the other messages specific to the components stay in the specific component, what we would import is `thoth.messaging.message_base` so we don't need to change thoth-messaging every time we modify only a specific component. WDYT? Or here we want to have one source for all messages? @KPostOffice @fridex @goern
> 
> I was wondering the same thing. Right now I have a naming convention for the messages so that it is obvious which package that the messages belong to. My only thought is if we end up needing a message in multiple repos then we'd have to reuse code between them.

So we keep everything in thoth-messaging related to messages and Kafka :)

File: thoth/messaging/__init__.py [hunk 1]
```diff
@@ -17,9 +17,11 @@
 
 """This is Thoth Messaging module."""
 
-from .package_releases import THOTH_PACKAGE_RELEASES_TOPIC_NAME, PackageRelease
-from .publish import KAFAK_BOOTSTRAP_SERVERS, KAFKA_CAFILE, create_topic, publish_to_topic
-
+from .message_base import MessageBase
```
Comment (pacospace) [inline thoth/messaging/__init__.py]: please add `from .message_base import  AdviseJustificationMessage` as well :)

File: thoth/messaging/advise_justification.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This is Thoth Messaging module for AdviseJustificationMessage."""
+
+
+import os
+import json
+import logging
+from typing import List
```
Comment (pacospace) [inline thoth/messaging/advise_justification.py]: This is not required, my bad :angel:

**Pull request merged**